tests/foreman/cli/test_satellitesync.py

```diff
@@ -439,17 +479,21 @@ def test_positive_export_import_redhat_cv(self):
             'id': exporting_cvv_id
         })
         exported_tar = '{0}/export-{1}-{2}.tar'.format(
-            self.export_base, self.exporting_cv_name, self.exporting_cvv_id)
+            self.export_base, rhel_cv_name, exporting_cvv_id)
         result = ssh.command("[ -f {0} ]".format(exported_tar))
         self.assertEqual(result.return_code, 0)
         exported_packages = Package.list({'content-view-version-id': exporting_cvv_id})
         self.assertTrue(len(exported_packages) > 0)
-        imp_rhel_repo = ContentViewSync._enable_rhel_content(self.importing_org, sync=False)
+        Subscription.delete_manifest({
```

Comment:
Was that extra space here intended?

Reply:
Nop, but Git showing it as extra space, it not in actual code. Btw, I had everything now on 1 line.